src/registrar/config/settings.py

```diff
@@ -95,6 +95,7 @@
 secret_registry_tenant_key = secret("REGISTRY_TENANT_KEY", None)
 secret_registry_tenant_name = secret("REGISTRY_TENANT_NAME", None)
 secret_registry_service_email = secret("REGISTRY_SERVICE_EMAIL", None)
+secret_dotgov_tenant_id = secret("DOTGOV_TEST_TENANT_ID", None)
```

Comment:
We use the test tenant for now. We'll need to gate this depending on prod vs. non-prod in the future.